### DISCUSSION TITLE
Extend my github education offers for a year

### DISCUSSION THREAD

**Discussion by Areg472** [community/community#145535] [GitHub Education] [answered]
### Hi everyone,

Hello. I'm a 9th grader in mkhitar sebastaci educational complex but I have only received my benefits until 11th grade, but I am gonna learn until 12th grade. Is it possible to extend the pack for a year? Thanks!

**Comment by florian-ariasu** [comment, marked as answer]
Aren t you able to renew it on this link: https://education.github.com/discount_requests/application. Carefully read the terms and conditions here.
